lib/getModelUploader.js

```diff
@@ -0,0 +1,24 @@
+const uc3100v1 = require('./models/uc3100-v1');
+const uc3100v2 = require('./models/uc3100-v2');
+const uc8100 = require('./models/uc8100');
+const compareVersions = require('./compareVersions');
+
+function getModelUploader(port) {
+  const { modelName, bootloaderVersion } = port.device;
+
+  if (/3100|3111|3121/.test(modelName)) {
+    if (compareVersions(bootloaderVersion, '1.3.0S07') >= 0) {
```

Comment:
@imZack This is the biggest question I have right now. Is the `1.3.0S07` good to use, or should we go to `1.4.0S00`?

Reply:
Once we get 1.4.0S00, we'll need to update this.